i18n/translations/en/form-builder.json

```diff
@@ -1070,13 +1070,17 @@
   "goBack": "Go back",
   "disabled": "disabled",
   "dynamicRow": {
-    "addButtonText": "Add",
+    "defaultRowTitle": "Row",
+    "rowTitleTextA11yEn": "Row title (english)",
+    "rowTitleTextA11yFr": "Row title (french)",
```

Comment:
I'm confused by this part: 

- What do we mean by "Row" here -- I'm not sure users will understand this new concept? Is a "row" the same as a "set", are they essentially whatever the global entity/object/category is?

Reply:
Ya not sure on the name --- in the markup it's the text that goes inside the fieldset legend.

Previously the default was "Item"

For each "Set" 

It will output 

Set 1
[fields]

Set 2
[fields] 

etc...

<img width="784" alt="368250971-dd4104d0-7ceb-481c-9ae1-86654fa870ec" src="https://github.com/user-attachments/assets/f9aa5cd9-eb03-420d-8035-5a68e86da5d9">